corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1488,7 +1490,10 @@ def __init__(self, report_name, app_id, source_type, report_source_id, existing_
             report_name, app_id, source_type, report_source_id, existing_report, *args, **kwargs
         )
         if self.source_type == "form":
-            self.fields['location'].widget = QuestionSelect(attrs={'class': 'input-large'})
+            self.fields['group_by'].widget = QuestionSelect(
```

Comment:
Should this be `self.fields['location'].widget`, considering this is a Map Report, or should line 1498 also be `self.fields['group_by'].widget`?